test/gtest/possible-contents.cpp

```diff
@@ -284,6 +284,13 @@ TEST_F(PossibleContentsTest, TestOracleMinimal) {
 void assertHaveIntersection(PossibleContents a, PossibleContents b) {
   EXPECT_TRUE(PossibleContents::haveIntersection(a, b));
   EXPECT_TRUE(PossibleContents::haveIntersection(b, a));
+#if BINARYEN_TEST_DEBUG
+  if (!PossibleContents::haveIntersection(a, b) ||
+      !PossibleContents::haveIntersection(b, a)) {
+    std::cout << "\nFailure: no intersection:\n" << a << '\n' << b << '\n';
+    abort();
+  }
+#endif
```

Comment:
Why do we need this? If this is the case, doesn't this crash in `EXPECT_TRUE` above anyway?

Reply:
`EXPECT_TRUE` doesn't print out the values being operated on, so we just get "expected true on line 286" and that's it. We don't even know which call to this function caused the problem. The extra logging makes it easy to debug these.